ingest/src/main/java/com/connexta/ingest/exceptions/StorageException.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.ingest.exceptions;
+
+public class StorageException extends RuntimeException {
+  public StorageException(String message, Throwable cause) {
```

Comment:
✏️ This doesn't need to be addressed in this PR but it'd be neat to have this extend a `RetryableException`

Reply:
`RetryableException` is actually a final class so we wouldn't be able to have a custom class extend it.